tests/conftest.py

```diff
@@ -27,9 +27,12 @@ def is_excluded_frontend_file(path):
 
 def pytest_addoption(parser):
     parser.addoption("--disable_sql_simplifier", action="store_true", default=False)
-    parser.addoption("--local_testing_mode", action="store_true", default=False)
     parser.addoption("--enable_cte_optimization", action="store_true", default=False)
     parser.addoption("--skip_sql_count_check", action="store_true", default=False)
+    if not any(
+        "--local_testing_mode" in opt.names() for opt in parser._anonymous.options
+    ):
```

Comment:
I'm curious what is this change for/what are `parser._anonymous.options`?

Reply:
`parser._anonymous.options` is where all of the options that have been added by the `addoptions` call are stored. This branch is in both `src/conftest.py` and `tests/conftest.py`. In some cases tox uses one conftest or the the other and in some it uses both so the option needs to be added at least once, but cannot be added more times without an error.

If both test locations shared a base path that we could add another the option in a shared conftest, but I don't think we want to add that to the basepath of the project.

Reply:
gotcha, thanks for the explanantion.